elastic/kibana
# [Fleet] Rely on package content during installation

## Summary

Closes https://github.com/elastic/kibana/issues/115032

Removing the call to `getInfo` in `Registry.getRegistryPackage` with `generatePackageInfoFromArchiveBuffer` to reduce the dependency from the registry.

### Tests

1. Try to install any package, it should work as usual
2. Run this script: `node x-pack/plugins/fleet/scripts/install_all_packages/index.js` - Remember to add the static path to the [kibana url](https://github.com/elastic/kibana/blob/4019d378f6e10884c699ba0e11035067b86eb9be/x-pack/plugins/fleet/scripts/install_all_packages/install_all_packages.ts#L13) if you use one:

![Screenshot 2022-10-03 at 18 29 38](https://user-images.githubusercontent.com/16084106/194040813-2f77ba02-3578-4ef7-aab2-13e96676631a.png)


### Checklist

Delete any items that are not applicable to this PR.
